src/views/Contacts.vue

```diff
@@ -145,7 +148,14 @@ export default {
 					},
 					text: group.name,
 					utils: {
-						counter: group.contacts.length
+						counter: group.contacts.length,
+						actions: [
+							{
+								icon: 'icon-download',
+								text: 'Download',
+								action: () => self.downloadGroup(group)
```

Comment:
Since you are using an arrow function, you should be able to use `this` right away instead of `self`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#No_separate_this

Did you try that? :)